geomstats/datasets/utils.py

```diff
@@ -179,3 +181,16 @@ def load_leaves():
     beta_param = gs.array(data[['nu', 'mu']])
     distrib_type = gs.squeeze(gs.array(data['Distribution']))
     return beta_param, distrib_type
+
+
+def load_emg():
```

Comment:
This function should be tested in: `tests/test_datasets.py`.

https://github.com/geomstats/geomstats/blob/master/tests/test_datasets.py

Maybe this will solve the decrease in code coverage found in the PR 🎉